perf/src/perf_libs.rs

```diff
@@ -146,6 +146,16 @@ pub fn append_to_ld_library_path(path: String) {
 }
 
 pub fn init_cuda() {
+    // force to ignore TEST_PERF_LIBS_CUDA and --cuda, to accomodate specific testing needs
+    if std::env::var("SOLANA_FORCE_NO_CUDA_FOR_CI").is_ok() {
```

Comment:
i know this is controversial...

but it's cumbersome to wire the no gpu flag from `ci/run-local.sh` all way down to here...

if this is unaccetable, I'll first commitin the other hunk.

Reply:
With https://github.com/solana-labs/solana/pull/18535, we can probably just skip `test-stable-perf` altogether in `run-local.sh` ?